.tekton/collector-build.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
-    pipelinesascode.tekton.dev/on-cel-expression: target_branch == "master" && event in ["push", "pull_request"]
+    pipelinesascode.tekton.dev/on-cel-expression: event in ["push", "pull_request"]
```

Comment:
Unfortunately, this needs one more iteration.
The reason is that we still need to have `target_branch` filter for `push` events. If we don't pushes to PR branches cause two builds: on `push` and one `pull_request`.
Builds should be identical because it's the same commit, the same pipeline and because we don't have any conditional logic on the even type. There could be potential issues from builds overwriting and/or appending duplicate data to the final artifacts but my concern is that it's just not the right thing to do.

It's hard to coordinate with that many PRs around. Let's sync on CEL expressions during today's meeting.